.github/workflows/test-ci-cd.yml

```
@@ -0,0 +1,120 @@
+
+      # 5. ECR 로그인
+      - name: Login to ECR
+        uses: aws-actions/amazon-ecr-login@v2
+
+      # 6. Docker 이미지 빌드 및 ECR 푸시
+      - name: Build, Tag and Push Docker Image to ECR
+        run: |
+          cd ${{ matrix.service.folder }}
+          docker build -t ${{ matrix.service.name }} .
+          docker tag ${{ matrix.service.name }}:latest ${{ secrets.ECR_REGISTRY }}/${{ matrix.service.name }}:latest
+          docker push ${{ secrets.ECR_REGISTRY }}/${{ matrix.service.name }}:latest      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)